database/migrations/2024_03_27_173140_create_webhooks.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('webhooks', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedInteger('campaign_id');
+            $table->unsignedTinyInteger('action');
+            $table->string('url', 191);
+            $table->unsignedTinyInteger('type');
+            $table->text('message')->nullable();
+            $table->timestamps();
+            $table->unsignedInteger('created_by')->nullable();
+            $table->unsignedInteger('updated_by')->nullable();
+            $table->unsignedTinyInteger('status');
+
+            $table->foreign('created_by')->references('id')->on('users')->onDelete('set null');
```

Comment:
onDelete('set null') can be replace by nullOnDelete() which is easier on the eyes